static/js/BookPage.jsx

```diff
@@ -245,8 +242,8 @@ class BookPage extends Component {
               <div className="tocTop">
                 <div className="tocTitle" role="heading" aria-level="1">
                   <div className="tocTitleControls">
-                    <ContentText text={{en:title, he:heTitle}}/>
-                    {moderatorSection}
+                    <CategoryHeader type="books" add_subcategory={false} add_section={true}
```

Comment:
as these are JS vars, we should use camelCase https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#object_names

Reply:
I see that snake_case is used throughout the code, it should be changed to camelCase which is the standard we use in JS